aliyun/src/main/java/org/apache/iceberg/aliyun/oss/OSSInputFile.java

```diff
@@ -24,17 +24,19 @@
 import org.apache.iceberg.exceptions.ValidationException;
 import org.apache.iceberg.io.InputFile;
 import org.apache.iceberg.io.SeekableInputStream;
+import org.apache.iceberg.metrics.MetricsContext;
 
+@Deprecated
```

Comment:
Can you add some context for this? Usually, we add javadoc to explain the deprecation:

```java
  /**
   * @deprecated moving to package-private in 0.15.0; use OSSFileIO to create InputFile instances.
   */
```

Reply:
Added the javadoc.